### PR TITLE
v2v:add TC 'Convert VMware ESX VM whose name contains percent encodin…

### PR DESCRIPTION
**Pull request by vwu-vera**
Please check bz#1938954

### CODE REVIEW

**Comment by vwu-vera** [comment]
This PR is related to #4457.

**Comment by vwu-vera** [comment]
@xiaodwan Please help to review. Thanks.:)

File: virttest/utils_v2v.py [hunk 1]
```diff
@@ -238,6 +238,9 @@ def _compose_vmx_filename():
                             ptn, self._nfspath).group(1)
                         break
 
+                if '%' in self._nfspath:
+                    self._nfspath = self._nfspath.replace('%', '%25')
+
```
Comment (xiaodwan) [inline virttest/utils_v2v.py:243]: I prefer to update the '_vmx_filename'.
self._vmx_filename = (self._vmx_filename + '.vmx').replace('%', '%25')

Comment (xiaodwan) [reply virttest/utils_v2v.py:243]: Ignore my previous comment. I was wrong. 
The _nfs_path is also used at other place.

**Review by xiaodwan — APPROVED** [review]

**Comment by xiaodwan** [comment]
@chunfuwen Could you help to review and merge this PR, please? thanks.

**Review by chunfuwen — APPROVED** [review]
LGTM

**Pull request merged**